ethereum/gnosis_safe/view_safes_eth_transfers.sql

```diff
@@ -0,0 +1,40 @@
+BEGIN;
+DROP MATERIALIZED VIEW IF EXISTS gnosis_safe.view_safes_eth_transfers;
+
+CREATE MATERIALIZED VIEW gnosis_safe.view_safes AS
```

Comment:
I would expect `view_safes_eth_transfers` here. Is that correct?

Reply:
🙈 Thank you. Fixed.